fhir-server/src/main/java/com/ibm/fhir/server/listener/FHIRServletContextListener.java

```diff
@@ -142,6 +143,10 @@ private void bootstrapDerbyDatabases(PropertyGroup fhirConfig) throws Exception
         Boolean performDbBootstrap = fhirConfig.getBooleanProperty(PROPERTY_JDBC_BOOTSTRAP_DB, Boolean.FALSE);
         if (performDbBootstrap) {
             log.info("Performing Derby database bootstrapping...");
+
+            // Start derby with debug off by default.
+            DerbyServerPropertiesMgr.setServerProperties(false);
```

Comment:
Is this actually needed?  Isn't "false" the default?

Or I guess you add this here just to make it easy to turn on should we want to turn it on?